language/move-lang/src/parser/syntax.rs

```diff
@@ -1360,6 +1360,50 @@ fn parse_field_annot<'input>(tokens: &mut Lexer<'input>) -> Result<(Field, Type)
     Ok((f, st))
 }
 
+//**************************************************************************************************
+// Address Block
+//**************************************************************************************************
+
+// Parse an address block:
+//      Address Block =
+//          "address" <Address> "{"
+//              <Module>*
+//          "}"
+fn parse_address_block<'input>(
+    tokens: &mut Lexer<'input>,
+) -> Result<(Loc, Address, Vec<ModuleDefinition>), Error> {
+    const UNEXPECTED_TOKEN: &str = "Invalid code unit. Expected 'address', 'module', or 'script'.";
```

Comment:
Most of our error diagnostics do not end with a period, do they? This should be consistent with the prevailing style.